components/StyledComponents/StyledPageContainer.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const StyledPageContainer = styled.div`
+  display: grid;
```

Comment:
Maybe this is a bit complicated to just put two element under each other? If you remove the container completely and give one of the children a padding, it will do the work. The default display is `block`, aligning elements left, under each other.

Reply:
you would need to remove the fixed position of the back button. this is the reason they are overlapping. Your solution is not wrong though